birdhouse/deployment/fix-geoserver-data-dir-perm

```diff
@@ -9,28 +9,26 @@
 # FIRST_RUN_ONLY, this script will always execute.
 
 THIS_FILE="$(readlink -f "$0" || realpath "$0")"
-THIS_DIR="$(dirname "$THIS_FILE")"
+THIS_DIR="$(dirname "${THIS_FILE}")"
+COMPOSE_DIR="${COMPOSE_DIR:-$(dirname "${THIS_DIR}")}"
 
-# Go to repo root.
-cd $THIS_DIR/../..
-
-. birdhouse/scripts/read-configs.include.sh
```

Comment:
FYI, location of scripts in this repo are part of the various candidate paths to try to auto-detect for `COMPOSE_DIR` in `detect_compose_dir` so that's why it was working without the variable `COMPOSE_DIR` set.

However, good idea to harmonize everything to serve as example for external scripts outside of this repo.

Reply:
Indeed. I think most of them were already covered by `discover_compose_dir`, but resolved using different approaches such as literal `../..` vs `$(dirname ...)`.